cmd/explaintest/main.go

```diff
@@ -278,6 +279,7 @@ func (t *tester) execute(query query) error {
 		return nil
 	}
 
+	collate.EnableNewCharset()
```

Comment:
It seems tidb will start with new charset feature, why enable it here?

Reply:
This is another program `explaintest`, which takes TiDB as a library.

Reply:
I think it is not, I found it execute SQL via `go-sql-driver/mysql`, looks like it start a tidb server